mojo-data/mojos/rulefit/missing/v1.00/rulefit_v1.00_missing_scores0.txt

```diff
@@ -0,0 +1,40 @@
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
+java.lang.ArrayIndexOutOfBoundsException
```

Comment:
this is a problem. this happens when during glm frame preparation there are some constant columns created and then they are dropped and mapping in mojo doesn't count with this situation and ends up with exception.